src/LightSpeed/toolbox.js

```diff
@@ -18,33 +20,46 @@ var LSToolbox = GObject.registerClass(class LSToolbox extends Toolbox {
 
         this._combinedTopic = new LSCombinedTopic();
         this._combinedTopic.bindGlobal(this._global);
-        this.addTopic('Game', 'rocket-symbolic', this._combinedTopic);
+        this.addTopic('game', 'Game', 'rocket-symbolic', this._combinedTopic);
 
         this._spawnEnemyTopic = new LSUserFunction('spawnEnemy');
         this._spawnEnemyTopic.bindGlobal(this._global);
-        this.addTopic('Enemies', 'astronaut-symbolic', this._spawnEnemyTopic);
+        this.addTopic('spawnEnemy', 'Enemies', 'astronaut-symbolic',
```

Comment:
Do you want me to wait to merge this until you have the correct icon for Enemies or do you want to put that in a different PR?

Reply:
No, I think we can replace the icons separately when the assets are delivered.